server/database/commands.go

```diff
@@ -0,0 +1,64 @@
+package database
+
+import (
+	"database/sql"
+
+	"github.com/rs/zerolog/log"
+)
+
+func InsertMoveCommand(db *sql.DB, id int64, commandType string, target int64) error {
+	log.Info().Msgf("insert into t_command (ship, type, target) values (%d, %s, %d)", id, commandType, target)
```

Comment:
command type is move. do not pass it as variable

Reply:
I don't want to hard code the exact name of the type in multiple places. This way, we only have it in the switch in stycoon/commands.go